### DISCUSSION TITLE
Syntax highlighting support for Ballerina.toml and other supported tomls

### DISCUSSION THREAD

**Discussion by rasika** [ballerina-platform/ballerina-lang#28146] [Ideas]
With the latest release of SL, we are supporting language features for the toml files including `Ballerina.toml`, `Dependencies.toml` and `Kubernetes.toml`.

@prabushi @hevayo @nadeeshaan @Kishanthan @hasithaa wouldn't it be nice to have syntax highlighting for highlighting language specific keywords through the VSCode plugin?

**Comment by nadeeshaan** [comment]
Currently there are several plugins to provide the syntax highlighting and more advanced features as well. May be we can use such extension as well (but we have to evaluate whether they break ballerina's toml experience). WDYT?

eg:
1. https://marketplace.visualstudio.com/items?itemName=be5invis.toml 
2. https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml
